spam/spam_tutorial.py

```diff
@@ -544,14 +670,14 @@ def plot_label_frequency(L):
 plot_label_frequency(L_train)
 
 # %% [markdown]
-# We see that over half of our training dataset data points have 0 or 1 weak labels.
-# Fortunately, the signal we do have can be used to train a classifier with a larger feature set than just these labeling functions that we've created, allowing it to generalize beyond what we've specified.
+# We see that over half of our training dataset data points have 2 or fewer labels from LFs.
+# Fortunately, the signal we do have be used to train a classifier with a larger feature set than just these labeling functions that we've created, allowing it to generalize beyond what we've specified.
 
 # %% [markdown]
-# ## 3. Combining Weak Labels with the Label Model
+# ## 3. Combining Labeling Functions with the Label Model
```

Comment:
We don't combine labeling functions, we intelligently combine/model/reweight the labels they produced

Reply:
lol i was torn by this...will make more precise